storage/innobase/xtrabackup/src/fil_cur.cc

```diff
@@ -242,6 +242,19 @@ xb_fil_cur_open(
 
 	/* Allocate read buffer */
 	cursor->buf_size = XB_FIL_CUR_PAGES * page_size;
+	if (opt_read_buffer_size != 0) {
+		ulint buf_size = cursor->buf_size;
+		const char *factors[] = {"b", "Kb", "Mb", NULL};
```

Comment:
Well, if it is done, it should be done globally. `innodb_log_file_size`, `use-mem` and many more can be expressed in KiBs MiBs and GiBs... I don't think it improves user experience a lot. It loses in precision however.